### PR TITLE
new content: testing

### PR DESCRIPTION
**Pull request by bulukucing**
@elianiva

### CODE REVIEW

File: content/membuat-vite-agar-mendukung-browser-lama.md [hunk 1]
```diff
@@ -66,3 +66,132 @@ export default {
...
+
+Disini saya akan melakukan uji coba
+untuk memastikan apakah website yang kita buat menggunakan vite
+akan tampil pada browser lama.
+
+Saya akan menggunakan Chrome android dengan versi 52 pada uji coba ini.
+
+![chrome android versi 52](https://raw.githubusercontent.com/bulukucing/assets/main/teknum-blog/22-01-01-chrome-android-versi-55.jpg)
+
+<p align="center" style={{fontSize: "0.875em"}}> Chrome android versi 55</p>
```
Comment (elianiva) [inline content/membuat-vite-agar-mendukung-browser-lama.md:83]:
```suggestion
<p align="center" style={{fontSize: "0.875em"}}> Chrome android versi 52</p>
```

File: content/membuat-vite-agar-mendukung-browser-lama.md [hunk 2]
```diff
@@ -66,3 +66,132 @@ export default {
...
+
+```bash
+npx vite build
+```
+
+Menggunakan Yarn:
+
+```bash
+yarn exec vite build
+```
```
Comment (elianiva) [inline content/membuat-vite-agar-mendukung-browser-lama.md:97]: yg kaya gini better digabung aja jadi satu code block

File: content/membuat-vite-agar-mendukung-browser-lama.md [hunk 3]
```diff
@@ -66,3 +66,132 @@ export default {
...
+
+```bash
+npm run build
+```
+
+Menggunakan Yarn:
+
+```bash
+yarn build
+```
```
Comment (elianiva) [inline content/membuat-vite-agar-mendukung-browser-lama.md:124]: ditto

File: content/membuat-vite-agar-mendukung-browser-lama.md [hunk 4]
```diff
@@ -66,3 +66,132 @@ export default {
...
+
+```bash
+npx vite preview --host
+```
+
+Menggunakan Yarn:
+
+```bash
+yarn exec vite preview --host
+```
```
Comment (elianiva) [inline content/membuat-vite-agar-mendukung-browser-lama.md:181]: ditto

**Comment by bulukucing** [comment]
sudah om

**Review by elianiva — APPROVED** [review]
thanks

**Pull request merged**